README.md

```diff
@@ -149,9 +150,11 @@ OR
     docker exec -it core_portal_cms /bin/bash
     python3 manage.py migrate
     python3 manage.py collectstatic --noinput
-    python3 manage.py createsuperuser
+    python3 manage.py createsuperuser  # Unless you will only login with your TACC account
+
+You may optionally create sample pages in the CMS.[^1]
 
-Finally, create a home page in the CMS.
+[^1]: First (if you have not already) [log in to the portal](https://cep.test/login). Then, [set your (automatically-created) CMS user as Staff and Superuser](https://github.com/TACC/Core-CMS/wiki/How-to-Set-Your-User-as-Staff-or-Superuser). Finally, follow instructions at https://cep.test/.
```

Comment:
> [!IMPORTANT]
> If you follow these steps, I expect adding a page to fail, like shown in [WP-897](https://tacc-main.atlassian.net/issues/WP-897). This PR is documentation, not bug fix.

Reply:
To me, this sounds like there will be instructions to follow on https://cep.test/. As far as I know, there aren't instructions there. Would it be more clear to say 

"Finally, create a home page in the CMS at https://cep.test/."?

Reply:
Ah, I see what you're saying! I think your "You may optionally.." sentence works great.

Reply:
K, kept new sentence but added at https://cep.test/, and removed footnote.